### PR TITLE
fix: fix decimal version bug (MAPCO-2765)

### PR DESCRIPTION
**Pull request by asafMasa**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: src/layers/models/layersManager.ts [hunk 1]
```diff
@@ -197,6 +198,14 @@ export class LayersManager {
     }
   }
 
+  /// validate productVersion will have decmal value
+  private validateCorrectProductVersion(data: IngestionParams): void {
+    // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+    if (data.metadata.productVersion?.indexOf('.') === -1) {
```
Comment (ronenkapelian) [inline src/layers/models/layersManager.ts:204]: should we protect case of N "." like "2.0.1" on case somebody provide such productVersion

File: src/layers/models/layersManager.ts [hunk 2]
```diff
@@ -197,6 +198,14 @@ export class LayersManager {
...
+      data.metadata.productVersion = `${data.metadata.productVersion}.0`;
+    }
+  }
+
   private validateGeoJsons(metadata: LayerMetadata): void {
     const footprint = metadata.footprint as Geometry;
     // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```
Comment (ronenkapelian) [inline src/layers/models/layersManager.ts:211]: maybe some unittest about the version nomrmalizing

File: src/layers/models/layersManager.ts (hunk 1 above)
Comment (asafMasa) [reply src/layers/models/layersManager.ts:204]: no as it won't pass the current regex  ^[1-9]\d{0,2}(\.(0|[1-9]\d?))?$

File: src/layers/models/layersManager.ts (hunk 2 above)
Comment (asafMasa) [reply src/layers/models/layersManager.ts:211]: added

**Review by ronenkapelian — APPROVED** [review]

**Pull request merged**